cmd/statusd/library.go

```diff
@@ -229,7 +229,7 @@ func CompleteTransaction(id, password *C.char) *C.char {
 	}
 	outBytes, err := json.Marshal(&out)
 	if err != nil {
-		log.Error("failed to marshal CompleteTransaction output", "error", err.Error())
+		log.Send(log.Errorf("failed to marshal CompleteTransaction output").With("error", err.Error()))
```

Comment:
Should we swap usage like:

```log.Send(log.Errorf("failed to marshal CompleteTransaction output").With("error", err))```

to 

```log.Error("failed to marshal CompleteTransaction output", log.Field{"error": err})``` ?

Reply:
I have been thinking about not changing the `log` API at all. It already has a good format:
```
log.Error("message", "tag1", value1, "tag2", value2, ...)
```
I believe you can convert it into JSON using a custom handler.

Reply:
Ya, spoke to @tiabc at length yesterday, will be reverting the log package back, and only keep changes in `goethereum` in `vendor`.